source/Halibut.Tests/Support/BackwardsCompatibility/LatestClientAndPreviousServiceVersionBuilder.cs

```diff
@@ -302,13 +303,17 @@ public TService CreateClient<TService>(Action<ServiceEndPoint> modifyServiceEndp
 
             public void Dispose()
             {
-                cancellationTokenSource?.Cancel();
-                Client.Dispose();
-                runningOldHalibutBinary.Dispose();
-                HttpProxy?.Dispose();
-                PortForwarder?.Dispose();
-                disposableCollection.Dispose();
-                cancellationTokenSource?.Dispose();
+                var logger = new SerilogLoggerBuilder().Build().ForContext<ClientAndService>();
+                logger.Information("Dispose called");
```

Comment:
Log when dispose is called, we suspect this is being called to early in some tests.